### PR TITLE
FE-267: fix custom analyzer & features not showing up in inverted index view, add support for basic fields

### PR DESCRIPTION
**Pull request by palashkaria**
### Scope & Purpose

- The way indices are noramalized was causing it to not show up when viewing  an inverted index. This fixes it by handling differently for disabled fields
- there are two ways of entering field details, this adds a check for when fields are an array of strings

- [x] :hankey: Bugfix

### Checklist

- [x] Tests
  - [x] Manually tested
- [x] :book: CHANGELOG entry made
- [x] Backports
  - [x] Backport for 3.11: https://github.com/arangodb/arangodb/pull/18950

#### Related Information

*(Please reference tickets / specification / other PRs etc)*

- [x] GitHub issue / Jira ticket: https://arangodb.atlassian.net/browse/FE-267

### CODE REVIEW

File: js/apps/system/_admin/aardvark/APP/react/src/views/collections/indices/addIndex/invertedIndex/FieldsDropdown.tsx [hunk 1]
```diff
@@ -41,8 +45,12 @@ const MultiValueLabel = (
 export const FieldsDropdown = ({ field }: { field: IndexFormFieldProps }) => {
   const [formikField] = useField<InvertedIndexValuesType[]>(field.name);
   const { setCurrentFieldData } = useInvertedIndexContext();
+  console.log("formikField", formikField.value);
```
Comment (nikhil-varma) [inline js/apps/system/_admin/aardvark/APP/react/src/views/collections/indices/addIndex/invertedIndex/FieldsDropdown.tsx:48]: Same here..

**Review by nikhil-varma — COMMENTED** [review]
Same question as the previous PR

**Review by nikhil-varma — APPROVED** [review]

**Review by ajurdak — APPROVED** [review]
LGTM.

File: js/apps/system/_admin/aardvark/APP/react/src/views/collections/indices/addIndex/invertedIndex/FieldsDropdown.tsx (hunk 1 above)
Comment (palashkaria) [reply js/apps/system/_admin/aardvark/APP/react/src/views/collections/indices/addIndex/invertedIndex/FieldsDropdown.tsx]: yeah, left it in by mistaske, removed now

**Comment by KVS85** [comment]
- [ ] https://jenkins01.arangodb.biz/job/arangodb-ANY-jslint/29032/

**Pull request merged**